modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,80 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by organization administrators, developers, and end users; each of these roles has different privileges when connecting applications,
+and there are some restrictions to ensure sensitive data remains secure.
+
+== Organization Administrator
+
+Organization administrators are the only users that can view and manage connected apps in Access Management.
+
+To manage your connected apps, navigate to Access Management and select the *Applications* tab.
```

Comment:
Note: I think the **Applications** tab will be renamed to **Connected Apps** in the release